### ISSUE TITLE
Inactive project

### ISSUE THREAD

**Issue by ktorn** [Technici4n/voxel-rs#13]
Any particular reason why this became inactive?
Just curious if there was any particular stumbling block that someone attempting a similar project should be aware of.

**Comment by Technici4n** [comment]
Hi! We wanted to write a playable Minecraft-like game, but it turns out that modding Minecraft is a lot easier because so much of the work is already done for you. We are happy with what we learned with voxel-rs, but both of us have moved on to Modern Industrialization and other Minecraft mods.

As for specific stumbling blocks, I would say that I find (modern) Java much more productive than Rust. Rust is a cool language, but I don't think it's suitable for this kind of large-scale project.

Overall if you want to have some insight into the workings of a Minecraft-like engine you should attempt such a project, but you should know that it will likely never be as good as Minecraft. If you want to create content and people to actually play with it, then modding Minecraft (or other games) is definitely the way to go.

**Comment by samuelh2005** [comment]
@Technici4n Could I borrow the engine and use it for my own game?
The current state of the engine is perfect for what I need to do with it.

**Comment by Technici4n** [comment]
Absolutely, just make sure to follow the (very permissive MIT) license. :)
